core/scripts/cli/cli_entry.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# This script should be installed with the UMA project and symlinked to the "uma" file in the npm binary directory.
+# Usage wherever the UMA package is installed:
+# $(npm bin)/uma --network mainnet_ledger
+
+# Platform independent way of evaluating symlinks to find the real location of the script.
+SCRIPT_FILE=$(python -c "import os; print(os.path.realpath(\"$0\"))")
```

Comment:
Perhaps over analyzing here, but you might want to specify python version

Reply:
Good point -- will do.

Reply:
Done.